Oneflare/dotfiles
# add tool-versions so we have a working default

We need to add some default versions so that we can run other tools as part of our repo setup scripts.

### How to test:
* Run `./install`
* You should now have a `~/.tool-versions` file
* Run `asdf current`
* Versions should match those in `tool-versions` and should be set by `/Users/<user>/.tool-versions`
